### PR TITLE
Added NegativeInt and NegativeFloat scalars

### PR DESCRIPTION
**Pull request by ccuilla**
Add a couple of simple scalars that were missed in the first version.

### CODE REVIEW

File: src/NegativeFloat.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  if (!(value < 0)) {
+    throw new TypeError(`Value is not a negative number: ${value}`);
+  }
+
+  return parseFloat(value);
+}
+
+export default new GraphQLScalarType({
+  name: 'PositiveFloat',
```
Comment (cfnelson) [inline src/NegativeFloat.js]: I think this name was meant to be `NegativeFloat` ?

File: src/NegativeInt.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  if (!(value < 0)) {
+    throw new TypeError(`Value is not a negative number: ${value}`);
+  }
+
+  return parseInt(value, 10);
+}
+
+export default new GraphQLScalarType({
+  name: 'PositiveInt',
```
Comment (cfnelson) [inline src/NegativeInt.js]: I believe `name` is meant to be `NegativeInt`?

File: src/NegativeFloat.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { GraphQLScalarType } from 'graphql';
+import { GraphQLError } from 'graphql/error';
+import { Kind } from 'graphql/language';
+
+function processValue(value) {
+  if (isNaN(value)) {
```
Comment (cfnelson) [inline src/NegativeFloat.js]: Unsure if this check is for detecting if the value passed is only a valid `number` or testing for `NaN`. Making an assumption based on the error message, maybe [Number.isFinite(value)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) would be better suited to validate that the value is not a Number. Assuming that you wish to disallow `NaN` & `Infinity`. 

Would recommend the changes for the other Number scalar types if the above assumption is correct.

File: src/NegativeFloat.js (hunk 1 above)
Comment (ccuilla) [reply src/NegativeFloat.js]: Good catch.

File: src/NegativeInt.js (hunk 2 above)
Comment (ccuilla) [reply src/NegativeInt.js]: Ditto.

File: src/NegativeFloat.js (hunk 3 above)
Comment (ccuilla) [reply src/NegativeFloat.js]: I'll check `isFinite`. I may also want to use `Number.isNaN()`. Thanks.

Comment (ccuilla) [reply src/NegativeFloat.js]: There are some other things I can do to maybe make this more robust like checking against `Number.MAX_SAFE_INTEGER`, etc. for example.

Comment (cfnelson) [reply src/NegativeFloat.js]: @ccuilla I realised that I should check the reference implementation as the `Number.isFinite()` will fail for strings. Check out how graphq.js are checking & coercing to [numbers](https://github.com/graphql/graphql-js/blob/master/src/type/scalars.js#L13).

File: package.json [hunk 4]
```diff
@@ -34,7 +34,7 @@
     "jest": "^20.0.4",
     "nodemon": "1.11.x"
   },
-  "dependencies": {
+  "peerDependencies": {
     "graphql": "^0.10.1"
```
Comment (cfnelson) [inline package.json]: @ccuilla You will want to also add `graphql` as a devDependencies as well to allow an simple quick install if someone clones your repo directly and wishes to run tests etc.

**Pull request merged**